website/server/libs/guildsAllowingBannedWords.js

```diff
@@ -0,0 +1,139 @@
+// Specify guilds that the automatic swearword blocker does not apply to:
+// - religious/atheist/philosophy guilds (legit use of words banned as oaths);
+// - food/drink/lifestyle/perfume guilds (alcohol allowed);
+// - guilds dealing with traumatic life events (must be allowed to describe them);
+// - foreign language guilds using the Roman alphabet (avoid accidental banning of non-English words).
+//
+// This is for a short-term, partial solution to the need for swearword blocking in guilds.
+// Later, it will be replaced with customised lists of disallowed words based on the guilds' tags.
+
+let guildsAllowingBannedWords = {
```

Comment:
I'm a bit concerned about this list, can we instead add an option to groups to allow banned words? It shouldn't take much and I can write the code an enable the option for the IDs below

Reply:
This is a short-term fix for rapid control of swear word blocking, until a proper solution is written and there's already a contributor interested in that issue. `guildsAllowingBannedWords` won't be in the code base permanently. I won't have time this week to make code changes and we'd really like to get this live soon. If you want to make all the changes that you're suggesting then I have no objections at all but I'm really not sure it's worth the effort. What concerns you about the list?

Reply:
that we'll forget about it but okay as otherwise there's no time for today's deploy to make the necessary changes :)

Reply:
Forgetting is a valid concern! I've added a note about deleting the temporary code here: https://github.com/HabitRPG/habitica/issues/8813#issuecomment-345061987